src/viam/components/movement_sensor/movement_sensor.py

```diff
@@ -147,6 +147,7 @@ async def get_readings(self, *, extra: Optional[Dict[str, Any]] = None, timeout:
                 compass: float,
                 orientation: Orientation,
             }
+
```

Comment:
should the below be removed? in `sensor.py` the only thing in `get_readings` is `...`. Should we not be calling everything manually now that we have the request/response to use?

Reply:
@njooma would definitely know. My guess is that now that you're not invoking `super().get_readings()`, you only need `...` here.

Reply:
yes we should remove the bespoke implementation in each sdk, we don't need it anymore. It will now be the responsibility of the driver using these wrappers.

Reply:
This was modeled after what's done in [rdk](https://github.com/viamrobotics/rdk/blob/12d225bb4da41d580576a44e8047bb5e9e88ce20/components/movementsensor/movementsensor.go#L106) where the base class for Movement Sensor defines the `Readings` function that all subclasses can use.

Reply:
@njooma, we added a common.proto `GetReadings` API, so we no longer have to do this in each SDK. Now we have a client and server-side implementation, and we're also changing the [go SDK](https://github.com/viamrobotics/rdk/pull/3047) implementation. So the way it's done now should be changed.